docker-compose.yaml

```
@@ -2,7 +2,9 @@ version: "3.4"
 
 services:
   api:
-    image: ghcr.io/amodeus-app/amodeus-api
+    # image: ghcr.io/amodeus-app/amodeus-api
+    build:
+      context: .
```

Let's leave it as-is for a while. I'll return to the project some day and rewrite the docker compose file entirely, also adjusting the API itself to the changes.

```suggestion
    image: ghcr.io/amodeus-app/amodeus-api
```